keyboards/walletburner/cajal/readme.md

```diff
@@ -0,0 +1,21 @@
+# Cajal
+
+![Cajal](https://i.imgur.com/FVKq7WH.png)
+
+The Cajal is a premium 45% keyboard with stagger or ortho layout options.
+
+* Keyboard Maintainer: [worldspawn00](https://github.com/worldspawn00/)
+* Hardware Supported: The Cajal PCB and kit.
+* Hardware Availability: [PCB & Kit](https://www.walletburner.co/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make walletburner/cajal:default
+
+    make walletburner/cajal:default_ortho
+
+Flashing example for this keyboard:
+
+    make walletburner/cajal:default:flash
+
```

Comment:
Can you add "clear instructions on how to reset the board into bootloader mode" as per [PR checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?

Reply:
I have asked @worldspawn00 about this, will update when I have the info.

Reply:
uh, "Push the reset button on the bottom of the PCB"

Reply:
Thanks @worldspawn00